web/src/lib/domain-logic/addAdditionalBusiness.test.ts

```diff
@@ -19,4 +19,33 @@ describe("addAdditionalBusiness", () => {
 
     expect(newUserData.businesses[newBusinessId].profileData).toEqual(createEmptyProfileData());
   });
+
+  it("adds a new empty business with matching userId", () => {
+    const userId = "multiple-biz-user-123";
+    const firstBusiness = generateBusiness({ userId: userId });
+    const userData = generateUserDataForBusiness(firstBusiness);
+    const newUserData = addAdditionalBusiness(userData);
+
+    expect(Object.keys(newUserData.businesses)).toHaveLength(2);
+    const newBusinessId = Object.keys(newUserData.businesses).find((id) => id !== firstBusiness.id)!;
+
+    expect(newUserData.businesses[newBusinessId].userId).toEqual(userId);
+  });
+
+  it("adds a new empty business with the same versionWhenAdded as the userData", () => {
+    const userId = "multiple-biz-user-123";
+    const firstBusiness = generateBusiness({
+      userId: userId,
+      versionWhenCreated: 110,
+    });
+    const userData = generateUserDataForBusiness(firstBusiness, {
+      versionWhenCreated: 110,
+    });
+    const newUserData = addAdditionalBusiness(userData);
+
+    expect(Object.keys(newUserData.businesses)).toHaveLength(2);
+    const newBusinessId = Object.keys(newUserData.businesses).find((id) => id !== firstBusiness.id)!;
+
+    expect(newUserData.businesses[newBusinessId].versionWhenCreated).toEqual(110);
+  });
```

Comment:
If we change how `business.versionWhenCreated` works, this test will likely need to change.